manual/source/Tutorial/DataStructures/Seeds.rst

```diff
@@ -44,16 +40,16 @@ In most cases, the :dox:`SimpleSeed Simple Seed` class is sufficient since the b
 
 You can get/set the begin and end position in the horizontal/vertical sequences using the functions :dox:`Seed#beginPositionH`, :dox:`Seed#beginPositionV`, :dox:`Seed#setBeginPositionH`, and :dox:`Seed#setBeginPositionV`.
 The band information can be queried and updated using :dox:`Seed#lowerDiagonal`, :dox:`Seed#upperDiagonal`, :dox:`Seed#setLowerDiagonal`, and :dox:`Seed#setUpperDiagonal`.
-Note, we use the capital letters 'H' and 'V' to indicate horizontal direction and vertical direction, respectively, while the database is always considered as the horizontal sequence and the query as the vertical sequence in the context of sequence alignments.
+Note, we use the capital letters 'H' and 'V' to indicate horizontal direction and vertical direction, respectively, while the **database** is always considered as the **horizontal sequence** and the **query** as the **vertical sequence** in the context of sequence alignments.
```

Comment:
I agree that "subject sequence" is a more general term.
However, I can find this much of "databases" in the current tutorial.
http://seqan.readthedocs.org/en/feature-manual-restructure/search.html?q=database&check_keywords=yes&area=default#

Should we consider them as well?